devfile.yaml

```diff
@@ -91,11 +96,23 @@ commands:
     actions:
       - workdir: /projects/src/github.com/eclipse/che-plugin-broker
         type: exec
-        command: './plugin-artifacts-broker --disable-push=true --runtime-id=workspace:developer:eclipse-che --registry-address=http://plugin-registry-local:8080 --metas ./brokers/testdata/config-plugin-ids.json'
+        command: >-
+          make build-artifacts &&
+          ./plugin-artifacts-broker \
+            --disable-push=true \
+            --runtime-id=workspace:developer:eclipse-che \
+            --registry-address=http://plugin-registry-local:8080 \
+            --metas ./brokers/testdata/config-plugin-ids.json
         component: dev
   - name: start plugin metadata broker
     actions:
       - workdir: /projects/src/github.com/eclipse/che-plugin-broker
         type: exec
-        command: './plugin-metadata-broker --disable-push=true --runtime-id=workspace:developer:eclipse-che --registry-address=http://plugin-registry-local:8080 --metas ./brokers/testdata/config-plugin-ids.json'
+        command: >-
```

Comment:
this command is not stable for me on che.openshift.io with persistent storage for some reason. I haven't tested with temporary storage.
![Screenshot_20200401_124758](https://user-images.githubusercontent.com/5887312/78124406-4d1ced80-7418-11ea-9d20-59822fca8b15.png)
![Screenshot_20200401_130132](https://user-images.githubusercontent.com/5887312/78124868-f82da700-7418-11ea-9653-05f74e26126d.png)

Reply:
*Strange*. We should figure out what that is happening before merge.